base/operators.jl

```diff
@@ -905,3 +905,7 @@ julia> map(splat(+), zip(1:3,4:6))
 ```
 """
 splat(f) = args->f(args...)
+
+anglebracket(x::Real, y::Real) = x * y
```

Comment:
isn't this redundant with the `dot` method?  Shouldn't this be defined in the LinearAlgebra package only?

Reply:
Should we leave some sort of dummy definition in Base? My vague impression is, if there is no `anglebracket` in Base, and PkgA and PkgB both defined and exported it, then there will be some kind of namespace clash? How to do it correctly?

Reply:
I think you can just do
`function anglebracket end`
which creates a function without any methods,

Reply:
Cool!